tests/test_base_device.py

```diff
@@ -1113,36 +1125,37 @@ def qiskit_circuit():
         qiskit_res = qiskit_circuit()
 
         assert np.shape(res) == np.shape(qiskit_res)
-        assert np.allclose(res, qiskit_res, atol=0.03)
 
-    def test_sampler_output_shape(self):
+    @pytest.mark.parametrize("backend", [backend, FakeManila(), FakeManilaV2()])
+    def test_sampler_output_shape(self, backend):
         """Test that the shape of the results produced from the sampler for the Qiskit device
         is consistent with Pennylane"""
-        dev = qml.device("default.qubit", wires=[0, 1, 2, 3], shots=1024)
-        qiskit_dev = QiskitDevice2(wires=[0, 1, 2, 3], backend=backend)
+        dev = qml.device("default.qubit", wires=5, shots=1024)
```

Comment:
Need to change number of wires to match the number of qubits for the fake hardware.